server/plugin/subscriptions.go

```diff
@@ -255,6 +265,66 @@ func (p *Plugin) StoreSubscriptions(s *Subscriptions) error {
 	return nil
 }
 
+func (p *Plugin) GetNotificationTurnedOffRepo() ([]string, error) {
```

Comment:
```suggestion
func (p *Plugin) GetExcludedNotificationRepos() ([]string, error) {
```
I think this name would be more intention revealing.

Reply:
Done